tests/cl-go-client/main.go

```diff
@@ -49,19 +59,22 @@ const (
 	denom1                          = "uusdc"
 	tickSpacing              int64  = 100
 	accountNamePrefix               = "lo-test"
-	numPositions                    = 100
-	numSwaps                        = 100
-	minAmountDeposited              = int64(1_000_000)
-	randSeed                        = 1
-	maxAmountDeposited              = 1_00_000_000
-	maxAmountSingleSwap             = 1_000_000
-	largeSwapAmount                 = 90_000_000_000
+	// Note, this is localosmosis-specific.
+	expectedEpochIdentifier = "hour"
```

Comment:
Are we going to run into a problem trying to use this for cl-testnet?

Reply:
The epoch identifier can manually be changed to "day" here.

A bigger problem is that we would have to wait for the epoch to pass for a while so that the incentive records are created

Reply:
I will try running this against testnet before I merge and document all the changes that would be necessary